oneview_redfish_toolkit/api/status_mapping.py

```diff
@@ -37,14 +37,65 @@
     }
 }
 
+HEALTH_STATUS_MAPPING = {
+    "Critical": "Critical",
+    "OK": "OK",
+    "Unknown": "Warning",
+    "Disabled": "Warning",
+    "Warning": "Warning",
+}
 
-def get_redfish_state(oneview_status):
-    """Gets corresponding Redfish State"""
+COMPOSITION_STATE_MAPPING = {
+    "NoProfileApplied": "Unused",
+    "ApplyingProfile": "Composing",
+    "ProfileApplied": "Composed",
+    "ProfileError": "Failed"
+}
+
+COMPUTER_SYSTEM_STATE_MAPPING = {
+    "NoProfileApplied": "Enabled",
+    "Monitored": "Enabled",
+    "ProfileApplied": "Enabled",
+    "Unsupported": "UnavailableOffline",
+    "Unknown": "Absent",
+    "RemoveFailed": "StandbyOffline",
+    "ProfileError": "StandbyOffline",
+    "Unmanaged": "StandbyOffline",
+    "Removing": "Updating",
+    "ApplyingProfile": "Updating",
+    "RemovingProfile": "Updating",
+    "UpdatingFirmware": "Updating",
+    "Adding": "Updating",
+    "Removed": "Disabled"
+}
+
+SERVER_PROFILE_STATE_MAPPING = {
+    "Normal": "Enabled",
+    "Creating": "Updating",
+    "Updating": "Updating",
+    "Deleting": "Updating",
+    "CreateFailed": "StandbyOffline",
+    "UpdateFailed": "StandbyOffline",
+    "DeleteFailed": "StandbyOffline"
+}
+
+
+def get_redfish_status_struct(oneview_status):
+    """Gets corresponding Redfish Status struct containing
 
-    return STATUS_MAP[oneview_status]["State"]
+        State and Health properties
+    """
+    state = STATUS_MAP[oneview_status]["State"]
+    health = STATUS_MAP[oneview_status]["Health"]
+    return state, health
+
+
+def get_redfish_state(oneview_state, ov_to_rf_map):
```

Comment:
I dont see so beneficits of using this method because all call must pass the status from oneview and choose the correct `ov_to_rf_map`, so the call could just excecute `ov_to_rf_map.get(oneview_state, None)` directly. Do you agree?!

Maybe could be better if we have other methods for the `ov_to_rf_map` be choosed inside these methods (without need the call choose externally the `ov_to_rf_map`).